README.md

```diff
@@ -60,12 +59,12 @@ Add the below additional configurations. Please verify these values and modify t
 
     zipkinExcludedUriList - uri list that is not needed for tracing
 
+    zipkinProxyType - zipkin proxy type i.e. Http, Socks4, Socks4a, Socks5 
+
 2) parameters.xml
```

Comment:
@kenyamat  removed parameters section. I agree that this would not be needed.
7444e9b
Thanks.